### PR TITLE
ACM Certificate region mask

### PR DESCRIPTION
**Pull request by ddollar**
## Release Playbook
- [ ] Rebase against master
- [ ] Release branch ()
- [ ] Pass CI ()
- [ ] Code review
- [ ] Merge into master
- [ ] Release master ()
- [ ] Pass CI ()
- [ ] Update staging rack
- [ ] Edit [Rack release record](https://github.com/convox/rack/releases) and/or update [docs](https://github.com/convox/convox.github.io)
- [ ] Publish release
- [ ] Release CLI

### CODE REVIEW

File: api/provider/aws/certificates.go [hunk 1]
```diff
@@ -147,6 +147,28 @@ func (p *AWSProvider) CertificateList() (structs.Certificates, error) {
 		})
 	}
 
+	// only fetch ACM certificates in regions that support it
+	switch os.Getenv("AWS_REGION") {
+	case "us-east-1":
+		c, err := p.certificateListACM()
+
```
Comment (nzoschke) [inline api/provider/aws/certificates.go]: @ddollar style tip... it seems like it's more idiomatic to put the error check immediately on the next line.  This helps another developer quickly see that errors are handled

Comment (ddollar) [inline api/provider/aws/certificates.go]: Hmm, not sure what you mean it's right here on line 155

Comment (nzoschke) [inline api/provider/aws/certificates.go]: no whitespace between the call and the error checking. line 154.

**Comment by nzoschke** [comment]
👍 releasing with https://github.com/convox/rack/pull/588

**Pull request closed without merging**